.eslintrc

```diff
@@ -1,11 +1,20 @@
 {
```

Comment:
Update the README.md on the TS migration guide/status? New scripts will need to add the migration require statement too